travis_pypi_setup.py

```diff
@@ -84,7 +85,7 @@ def load_yaml_config(filepath):
 
 
 def save_yaml_config(filepath, config):
-    with open(filepath, 'w') as f:
+    with open(filepath, "w") as f:
```

Comment:
[inline-plz]: `prospector: pylint: Variable name "f" doesn't conform to snake_case naming style (invalid-name)`